source/manual/alerts/rabbitmq-no-consumers-listening.html.md

```diff
@@ -4,75 +4,106 @@ title: 'RabbitMQ: No consumers listening to queue'
+
+### Check the RabbitMQ logs
+
+* Check the logs for the `rabbitmq` application which can be achieved logging
+  into Kibana and searching for `application: rabbitmq`. Is there evidence of
+  any errors?
+
+### Check the RabbitmQ Grafana dashboard
+
+This [Grafana dashboard][rabbitmq_grafana_dashboard] shows activity across
+multiple exchanges and queues. The main exchange we expect to be monitoring is
+`published_documents` which handles broadcasting to services such as search and
+email-alert-service when content changes across GOV.UK.
+
+Looking at the queue graphs we should look out for the following:
+
+* Check for high 'ready' messages for the queue
+
+This indicates these messages are waiting to be processed in RabbitMQ by the
+consumer (e.g email-alert-service).
+
+* Check for high 'unacknowledged' messages for the queue
+
+This implies that messages have been read by the consumer but the consumer has
+never sent back an ACK to the RabbitMQ broker to say that it has finished
+processing.
+
+* Check for high 'redeliver' rate for the queue
 
-## Consequences of idle consumers
+In the event of network failure (or a node failure), messages can be
+redelivered. An example is if the consumer dies (its channel is closed,
+connection is closed, or TCP connection is lost).
 
-If consumers are idle, messages sent to the queue will not be being processed.
-This means that the actions which are meant to happen in response to the
-messages will not happen. For example, email alerts about updated content will
-not be sent if the `email_alert_sevice` queue isn't running.
+If we're seeing high 'redeliver' rates, high 'ready' or 'unacknowledged'
+messages then this could indicate an issue with the consumer.
 
-Unless there is a wider RabbitMQ failure, messages will not be lost - they will
-be processed once the problem is resolved.
+### Troubleshooting steps
 
-## Investigation
+1. Check the status of the consumer, e.g for email-alert-service look on the
+   email_alert_api machines - `sudo service email-alert-service status`
 
-RabbitMQ has an admin interface, which allows details of recent activity on the
-queues to be seen. To log in, [follow the instructions in the RabbitMQ
-docs][rabbitmq_control_panel].
+2. If the services are running we should look in the consumers application logs.
 
-The admin interface allows you to see details (and graphs!) of the messages
-sent to each queue, and the number of messages held on the queues. Look at the
-queue for which the alert happened. If no messages have been sent to the queue
-over the last several minutes, the most likely failure is that the heartbeat
-messages are no longer being sent correctly. Look for recent changes to the
-[Publishing API][publishing_api].
+3. If messages are being received by the consumer check they are actually being
+   actioned (e.g email-alert-api is receiving the messages email-alert-service
+   has received from RabbitMQ). To do this we could compare the content changes
```

Comment:
This is too specific, given the alert applies to several apps. I think we should see errors in all cases, somewhere, so we can just stick with the previous step.